week3/Week 3 Playground.playground/Contents.swift

```diff
@@ -0,0 +1,123 @@
+import SwiftUI
+
+// a)
```

Comment:
```swift
let nums = Array(0...20)

for number in nums {
    if number % 2 == 0 {
        print(number)
    }
}

Or

for num in nums where num.isMultiple(of: 2) { 
  print(i) 
}
```

Reply:
Oh, the `isMultiple(of: )` method looks super useful, I’ll keep that one in mind. Thanks!